database/migrations/2016_12_01_210658_create_events_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateEventsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('events', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('northstar_id')->index()->comment('The users Northstar ID');
+            $table->string('event_type')->index()->comment('the type of event');
```

Comment:
@angaither wait `event_type` does have an index on it. are you saying to remove it? or keep it?